.env.development

```diff
@@ -57,3 +57,4 @@ PLOTLY_SERVER_URL='http://localhost:8050'
 AUTH0_SELF_SERVICE_INTEGRATION='true'
 MFE_CONFIG_API_URL='http://localhost:18000/api/mfe_config/v1'
 DEMO_ENTEPRISE_UUID='set a valid enterprise uuid'
+EDX_ACCESS_URL='http://2u-guid-dev.us.auth0.com'
```

Comment:
I guess we should set it empty here considering its opensource and `http://2u-guid-dev.us.auth0.com` will not be available to everyone.